README.md

```diff
@@ -1,22 +1,41 @@
-Raincat
+# Raincat
 Game Creation Society - Fall 08
-www.gamecreation.org
 
+https://www.gamecreation.org/
 
-HOW TO START THE GAME:
-----------------------
-./raincat
+### Installation (Snap Store)
+Raincat is available in the Snap Store at: https://snapcraft.io/raincat-game
+
+It can be installed with:
+```
+sudo snap install raincat-game
+```
+
+After installation, Raincat can be launched from the .desktop launcher, or with:
+```
+snap run raincat-game.raincat
```

Comment:
Probably `raincat-game.desktop`

Reply:
Sorted this to reference the `raincat-game.desktop` file directly. I didn't bother as the Snap installation procedure puts it in the right XDG Application directory to have a proper launcher for desktop environments.

Reply:
I mean shouldn't it be `snap run raincat-game.desktop` instead of `snap run raincat-game.raincat`

Reply:
Ah I see what you mean. This is an odd quirk of how Snapcraft handles binaries within the Snap container.

In an ideal world

Reply:
I just get access to Raincat name on snap. Could you please update your PR? And thanks for your explanation, a lot of new info.

Reply:
> I just get access to Raincat name on snap.

Ah awesome :smile:. I pushed a new commit just now which should update things to use the `raincat` name. That's much cleaner!